### ISSUE TITLE
When using docker-compose, `Remote-Containers: Rebuild Container` doesn't actually rebuild the container

### ISSUE THREAD

**Issue by smiller171** [microsoft/vscode-remote-release#564]
Issue Type: <b>Bug</b>

1. Set up a remote container using docker-compose with the `build` directive
2. Launch workspace in container
3. change Dockerfile
4. run `Remote-Containers: Rebuild Container`

Expected behavior: Container is rebuilt
Actual behavior: Cached container is used
Solution : Call `docker-compose` with the `--build` directive when running the `Rebuild Container` command

Extension version: 0.58.0
VS Code version: Code - Insiders 1.35.0-insider (553cfb2c2205db5f15f3ee8395bbd5cf066d357d, 2019-06-04T00:11:00.788Z)
OS version: Darwin x64 18.6.0
Remote OS version: Linux x64 4.9.125-linuxkit

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz (8 x 2300)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|6, 7, 8|
|Memory (System)|8.00GB (0.08GB free)|
|Process Argv|-psn_0_4129776|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|Dev Container: Go|
|OS|Linux x64 4.9.125-linuxkit|
|CPUs|Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz (2 x 2300)|
|Memory (System)|0.97GB (0.11GB free)|
|VM|0%|
</details>

**Comment by smiller171** [comment]
looks like you can use `runArgs` to accomplish this, but `runArgs` is documented as only working for `Dockerfile` or `Image`. It would seem that it should be moved to `general` in the docs

**Comment by chrmarti** [comment]
Could you append your devcontainer.json? How did you connect to the container, with 'Reopen Folder in Container'?

**Comment by smiller171** [comment]
I actually switched back away from using compose since I couldn't make the socket mounts I was trying to use work the way I wanted to, but any `devcontainer.json` using docker-compose would work. I originally connected with "Reopen Folder in Container", then made a change to the Dockerfile and used "Rebuild Container" from the containerized window. This relaunched the workspace on the same image instead of rebuilding the image.

I was able to use `"runArgs": ["--build"]` to get it to rebuild the image, but the `runArgs` flag is documented as being for images and dockerfiles only, not docker-compose
